src/scripts/main.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
 const people = require('./lib/people');
+const tableOfPeople = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const { name, sex, born, died } of people) {
+  const tableRow = document.createElement('tr');
 
-// write your code here
+  tableRow.innerHTML = `
+  <td>${name}</td>
+  <td>${sex === 'f'
+    ? 'Female'
+    : 'Male'}
+  </td>
```

Comment:
```suggestion
  <td>
    ${
      sex === 'f'
        ? 'Female'
        : 'Male'
    }
  </td>
```
Чесно не знаю як правильно але так як ти зробив також здається не ок, краще взагалі не переносити))